### DISCUSSION TITLE
Release requested for #677 changeset

### DISCUSSION THREAD

**Discussion by jonnybbb** [cloudbees/zendesk-java-client#700] [General]
Are there plans to make a new release as I would like to use the changes out of  #677

**Comment by PierreBtz** [comment]
Sorry, this changeset went under my radar, I'll perform a release this week.

**Comment by PierreBtz** [comment]
Release done, sorry again for the delay...

https://github.com/cloudbees-oss/zendesk-java-client/discussions/705
